logdna/resource_member_test.go

```diff
@@ -0,0 +1,68 @@
+package logdna
+
+import (
+	"regexp"
+	"strings"
+	"testing"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource"
+)
+
+func TestMember_ErrorRoleEmpty(t *testing.T) {
+	args := map[string]string{
+		"email": `"sampleemail@gmail.com"`,
```

Comment:
nit: we should avoid using real emails/domains in tests, and prefer using the `example.org` domain which is [reserved](https://www.iana.org/domains/reserved) for this purpose.

same for the below cases unless we actually need a real mezmo.com address for these integration tests.

Reply:
Fixed